src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -52,13 +53,32 @@ internal class BindingProcessImpl : IBindingProcess
             SonarLintMode bindingMode,
             IExclusionSettingsStorage exclusionSettingsStorage,
             bool isFirstBinding = false)
+            : this(host,
+                  bindingArgs,
+                  solutionBindingOperation,
+                  bindingConfigProvider,
+                  bindingMode,
+                  exclusionSettingsStorage,
+                  isFirstBinding,
+                  languagesToBind: Language.KnownLanguages)
+        { }
+
+        internal /* for testing */ BindingProcessImpl(IHost host,
+            BindCommandArgs bindingArgs,
+            ISolutionBindingOperation solutionBindingOperation,
+            IBindingConfigProvider bindingConfigProvider,
+            SonarLintMode bindingMode,
+            IExclusionSettingsStorage exclusionSettingsStorage,
+            bool isFirstBinding,
+            IEnumerable<Language> languagesToBind)
```

Comment:
Added an internal constructor so we can inject the list of languages when testing.